submarine-sdk/pysubmarine/submarine/store/model_registry/sqlalchemy_store.py

```diff
@@ -0,0 +1,570 @@
+                    source=source,
+                    user_id=user_id,
+                    experiment_id=experiment_id,
+                    creation_time=creation_time,
+                    last_updated_time=creation_time,
+                    dataset=dataset,
+                    description=description,
+                    model_tags=[SqlModelTag(tag=tag) for tag in tags or []],
+                )
+                self._save_to_db(session, [sql_registered_model, model_version])
+                session.flush()
+                return model_version.to_submarine_entity()
+            except sqlalchemy.exc.IntegrityError:
+                raise SubmarineException(message=f"Model Version creation error (name={name}).")
+
+    @classmethod
+    def _get_model_version(
+        cls, session: Session, name: str, version: int, eager: bool = False
+    ) -> SqlModelVersion:
+        """
+        :param eager: If ``True``, eagerly loads the model version's tags.
+                      If ``False``, these attributes are not eagerly loaded and
+                      will be loaded when their corresponding object properties
+                      are accessed from the resulting ``SqlModelVersion`` object.
+        """
+        validate_model_name(name)
+        validate_model_version(version)
+        query_options = cls._get_eager_model_version_query_options() if eager else []
+        conditions = [
+            SqlModelVersion.name == name,
+            SqlModelVersion.version == version,
+            SqlModelVersion.current_stage != STAGE_DELETED_INTERNAL,
+        ]
+
+        versions: List[SqlModelVersion] = (
+            session.query(SqlModelVersion).options(*query_options).filter(*conditions).all()
+        )
+        if len(versions) == 0:
+            raise SubmarineException(f"Model Version (name={name}, version={version}) not found.")
+        elif len(versions) > 1:
+            raise SubmarineException(
+                f"Expected only 1 model version with (name={name}, version={versions}). Found"
+                f" {len(versions)}."
+            )
+        else:
+            return versions[0]
+
+    def update_model_version_description(
+        self, name: str, version: int, description: str
+    ) -> ModelVersion:
+        """
+        Update description associated with a model version in backend.
+        :param name: Registered model name.
+        :param version: Registered model version.
+        :param description: New model description.
+        :return: A single :py:class:`submarine.entities.model_registry.ModelVersion` object.
+        """
+        with self.ManagedSessionMaker() as session:
+            update_time = datetime.now()
+            sql_model_version = self._get_model_version(session, name, version)
+            sql_model_version.description = description
+            sql_model_version.last_updated_time = update_time
+            self._save_to_db(session, sql_model_version)
```

Comment:
Is it necessary to check the length of description?

Reply:
I will add the check